scraping/spiders/mapujpomoc.py

```diff
@@ -0,0 +1,141 @@
+import logging
+
+import scrapy
+
+from config.constants import DEFAULT_CATEGORY
+
+
```

Comment:
I think we could address some of the unclassifiedcategories like so:

* Afterschool activities for children -> Children
* Counteracting sexual violence and human trafficking -> Social help
* Flashlights, batteries, chargers, power banks -> Electronics
* Polish language classes -> Education
* Training workshops for teachers or teaching assistants -> Education